saleor/core/models.py

```diff
@@ -47,7 +48,9 @@ def published(self):
 
     @staticmethod
     def user_has_access_to_all(user):
-        return user.is_active and user.has_perm("product.manage_products")
+        return user.is_active and user.has_perm(
+            ProductPermissions.MANAGE_PRODUCTS.value
```

Comment:
I thought about this,
`has_perm` returns list of string (which is extracted from Permission model), I would have to change this to return list of enums
and because we don't have one PermissionEnum I would have to create function that would take every permission enum and check if given codename is in enum
and then create list out of it.

imho too much work, and make code less readable, and in makes maintenance more difficult

Reply:
changed enum.value to enum